support/src/traits.rs

```diff
@@ -80,3 +85,50 @@ pub trait GetWorstCase {
 pub trait ItemFilter<Item> {
 	fn should_include(&self, credential: &Item) -> bool;
 }
+
+pub trait StorageDepositCollector<AccountId, Key> {
+	type Currency: ReservableCurrency<AccountId>;
+
+	fn deposit(
```

Comment:
S: Add docs for all exposed functions

Reply:
[f1ef9ca](https://github.com/KILTprotocol/kilt-node/pull/427/commits/f1ef9ca483f68deb53716b38cb95482b00ae764b)